app/src/main/java/com/example/firstapplication/ToDo.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.firstapplication
+
+data class ToDo(
+    val id: String,
+    val userId: String,
+    val title: String?,
+    var completed: Boolean
```

Comment:
Mutowalne pole. Jego wartość jest zmieniana w adapterze.
Ciekawe :) Osobiście bałbym się tak zrobić, bo mechanizm szukający różnic między nową a starą listą (diff util) działa w osobnym wątku, więc jeśli zmodyfikujesz tę wartość w czasie, gdy ten mechanizm pracuje, może on wyliczyć nieprawidłowe wyniki. Z drugiej strony gdyby klasa `ToDo` miałaby być niemutowalna, musiałabyś napisać trochę więcej kodu by obsłużyć zmianę w modelu. Tak, jak masz, jest prosto, i jeśli nie widzisz problemów w działaniu, to zostawmy :)